From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 20 Mar 2026 01:16:56 +0100
Subject: shell: Remove set-but-unused variable

The variable is set, but never read. Indeed, it does not exist
in the original Firefox code the method is based on.

Remove it to shut up a new gcc warning (-Wunused-but-set-variable).

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/4142>
Origin: upstream, 50.1, commit:ecb7dcc5d67c33d2d5155ecfc32fc36a7e4cfa08
---
 src/shell-keyring-prompt.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/shell-keyring-prompt.c b/src/shell-keyring-prompt.c
index 9a518c8..821505b 100644
--- a/src/shell-keyring-prompt.c
+++ b/src/shell-keyring-prompt.c
@@ -601,7 +601,7 @@ shell_keyring_prompt_get_confirm_actor (ShellKeyringPrompt *self)
 static guint
 calculate_password_strength (const gchar *password)
 {
-  int upper, lower, digit, misc;
+  int upper, digit, misc;
   gdouble pwstrength;
   int length, i;
 
@@ -619,7 +619,6 @@ calculate_password_strength (const gchar *password)
     return 0;
 
   upper = 0;
-  lower = 0;
   digit = 0;
   misc = 0;
 
@@ -627,8 +626,6 @@ calculate_password_strength (const gchar *password)
     {
       if (g_ascii_isdigit (password[i]))
         digit++;
-      else if (g_ascii_islower (password[i]))
-        lower++;
       else if (g_ascii_isupper (password[i]))
         upper++;
       else
